docs/Hosting_Graphspace.md

```diff
@@ -0,0 +1,13 @@
+# Hosting GraphSpace
+
+## How to deploy the latest Github code changes on GraphSpace server? ##
+
+Here are the steps for deploying GraphSpace. It is recommended to download the [latest release](https://github.com/Murali-group/GraphSpace/releases/latest). Otherwise you can download the latest code directly from the [master branch](https://github.com/Murali-group/GraphSpace).
+Then follow the steps to [run GraphSpace on Apache](https://github.com/Murali-group/GraphSpace#running-graphspace-on-apache). These include steps to install the correct version of python, setup postgreSQL, virtualenv, bower, ElasticSearch, etc.
+
+>**Note**:
+>Be careful when running alembic migration scripts if you plan to continually update a deployment of GraphSpace. Understand how to use [alembic](https://alembic.sqlalchemy.org/en/latest/tutorial.html#) and alembic commands. Migrations are not needed if you are creating the database for the first time.

```

Comment:
There are no instructions on how to update an existing deployment of GraphSpace on a server. Shouldn't these instructions appear in that section?

Reply:
If someone is updating their GraphSpace with an update that includes migrations, they should be running standard alembic commands that aren't specific to GraphSpace itself. I can add more docs about that but I think the link to alembic and I added the code to 'upgrade to newest migration'.